test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -231,23 +243,27 @@ public void GetTableDefinitionInvalidObjectTest()
         [Fact]
         public void GetTableDefinitionWithSchemaTest()
         {
-            // Get live connectionInfo
-            PeekDefinition peekDefinition = new PeekDefinition(TestObjects.InitLiveConnectionInfoForDefinition());
+            // Get live connectionInfo and serverConnection
+            ConnectionInfo connInfo = TestObjects.InitLiveConnectionInfoForDefinition();
+            ServerConnection serverConnection = TestObjects.InitLiveServerConnectionForDefinition(connInfo);
+
+            PeekDefinition peekDefinition = new PeekDefinition(serverConnection, connInfo);
             string objectName = "spt_monitor";
+
             string schemaName = "dbo";
             string objectType = "TABLE";
 
             // Get locations for valid table object with schema name
             Location[] locations = peekDefinition.GetSqlObjectDefinition(peekDefinition.GetTableScripts, objectName, schemaName, objectType);
             Assert.NotNull(locations);
-            Cleanup(locations);            
+            Cleanup(locations);
         }
 
         /// <summary>
-        /// Test GetDefinition with an unsupported type(function)
+        /// Test GetDefinition with an unsupported type(function). Expect a error result.
         /// </summary>
         [Fact]
-        public void GetUnsupportedDefinitionForFullScript()
+        public void GetUnsupportedDefinitionErrorTest()
         {
 
```

Comment:
Minor: please add a `// given I am querying an unsupported object (Schema)` to match given/when/then pattern and make clear that it's the dbo schema you are getting the definition for